homeassistant/components/cast/media_player.py

```diff
@@ -185,7 +186,8 @@ async def async_added_to_hass(self):
         )
         self.hass.bus.async_listen_once(EVENT_HOMEASSISTANT_STOP, self._async_stop)
         self.async_set_cast_info(self._cast_info)
-        self.hass.async_create_task(
+        # asyncio.create_task is used to avoid delaying startup wrapup
```

Comment:
```suggestion
        # asyncio.create_task is used to avoid delaying startup wrapup if the device 
        # is discovered already during startup but then fails to respond
```